docs_sources/using.md

```diff
@@ -517,3 +517,8 @@ When using the command-line interface, the configuration file specified in the `
 For more detailed examples of how to customize TPOT's operator configuration, see the default configurations for [classification](https://github.com/rhiever/tpot/blob/master/tpot/config_classifier.py) and [regression](https://github.com/rhiever/tpot/blob/master/tpot/config_regressor.py) in TPOT's source code.
 
 Note that you must have all of the corresponding packages for the operators installed on your computer, otherwise TPOT will not be able to use them. For example, if XGBoost is not installed on your computer, then TPOT will simply not import nor use XGBoost in the pipelines it considers.
+
+<a name="usagenotes"></a>
+## Usage Notes
```

Comment:
Maybe change this to "Missing values"

and add a note that if the user doesn't want to use median imputation, then they should apply their own imputation method prior to passing the data to TPOT.